Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -29,10 +29,6 @@
 #:    If `--message=`<message> is passed, append <message> to the default PR
 #:    message.
 #:
-#:    If `--no-browse` is passed, don't pass the `--browse` argument to `hub`
```

Comment:
Are we going to replace the browser functionality?

Reply:
Do we want to ? I thought that browser functionality was due to `hub`. 
Its possible to preserve that functionality if required.

Reply:
The browser functionality is due to hub, but hub is doing it to attempt to be user friendly. You just opened a PR so presumably you want to inspect the final PR in your browser. The existing bump-formula-pr behavior is to print the PR url if --no-browse was passed, otherwise open the PR in the user's browser. I do think we should at least be printing the PR's url.

@MikeMcQuaid any strong feelings about the Terminal.app-to-Safari.app 🚀 ?

Reply:
Can use `open` to open URLs with default handler. Use the `PATH_OPEN` constant.

Reply:
Using `exec_browser` method defined in `utils` to open the browser.